benchmarks/dgemm_compare.py

```diff
@@ -48,9 +48,10 @@
             b_new = b
             c_new = c
 
-        view_a = pk.from_numpy(a_new)
-        view_b = pk.from_numpy(b_new)
-        view_c = pk.from_numpy(c_new)
+        view_a = pk.from_array(a_new)
```

Comment:
Do we need from_array or can we just say from?

Reply:
`array` is fine, but having one is way better than having three

Reply:
I'm fine with hiding all the rest so we only have one interface... just wasn't sure if we wanted to keep the backwards compatibility or not. I like array(), if we are hiding the others, I think it makes sense to be as numpy-like as possible. I only used from_array() to match the existing functions.

Reply:
let's do `array` and hide others